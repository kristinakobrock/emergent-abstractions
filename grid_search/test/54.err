  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.99it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.11it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 22.61it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 26.76it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 27.58it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 27.46it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 25.45it/s] 50%|##############################################                                              | 25/50 [00:00<00:01, 24.41it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 23.75it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 25.23it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 24.29it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 23.50it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 23.14it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 24.85it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 23.99it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 25.53it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.15it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.34it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 29.72it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 29.79it/s]100%|############################################################################################| 13/13 [00:00<00:00, 28.14it/s]100%|############################################################################################| 13/13 [00:00<00:00, 28.68it/s]
