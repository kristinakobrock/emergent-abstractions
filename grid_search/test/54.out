# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.694381058216095, "acc": 0.5940961241722107, "length": 1.8121198415756226, "mode": "train", "epoch": 1}
{"loss": 0.6673233509063721, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6620917320251465, "acc": 0.6261693835258484, "length": 1.0113232135772705, "mode": "train", "epoch": 2}
{"loss": 0.6612194776535034, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6613755822181702, "acc": 0.6257066130638123, "length": 1.005798101425171, "mode": "train", "epoch": 3}
{"loss": 0.6645834445953369, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6615192294120789, "acc": 0.6260530948638916, "length": 1.003465175628662, "mode": "train", "epoch": 4}
{"loss": 0.6670098900794983, "acc": 0.6184027194976807, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6617954969406128, "acc": 0.6261675357818604, "length": 1.002604603767395, "mode": "train", "epoch": 5}
{"loss": 0.6618561744689941, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6610010862350464, "acc": 0.6266229748725891, "length": 1.0018662214279175, "mode": "train", "epoch": 6}
{"loss": 0.6648591160774231, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6634596586227417, "acc": 0.6236562728881836, "length": 1.0014832019805908, "mode": "train", "epoch": 7}
{"loss": 0.6681998372077942, "acc": 0.6215277910232544, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6615269780158997, "acc": 0.6251973509788513, "length": 1.0011606216430664, "mode": "train", "epoch": 8}
{"loss": 0.6632411479949951, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6587041020393372, "acc": 0.6262254118919373, "length": 1.0008124113082886, "mode": "train", "epoch": 9}
{"loss": 0.6646201014518738, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6587133407592773, "acc": 0.6261684894561768, "length": 1.0007468461990356, "mode": "train", "epoch": 10}
{"loss": 0.6629256010055542, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6611217260360718, "acc": 0.6258823871612549, "length": 1.000415563583374, "mode": "train", "epoch": 11}
{"loss": 0.6651482582092285, "acc": 0.621874988079071, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.659795343875885, "acc": 0.6251989603042603, "length": 1.0003352165222168, "mode": "train", "epoch": 12}
{"loss": 0.6653869152069092, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6603617072105408, "acc": 0.6260547041893005, "length": 1.0007590055465698, "mode": "train", "epoch": 13}
{"loss": 0.6635036468505859, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6588656306266785, "acc": 0.6265109181404114, "length": 1.0001418590545654, "mode": "train", "epoch": 14}
{"loss": 0.6635172367095947, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6590037941932678, "acc": 0.626339852809906, "length": 1.000090479850769, "mode": "train", "epoch": 15}
{"loss": 0.6631913185119629, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6592676043510437, "acc": 0.6262256503105164, "length": 1.0000641345977783, "mode": "train", "epoch": 16}
{"loss": 0.6634389162063599, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6585508584976196, "acc": 0.6262260675430298, "length": 1.0000940561294556, "mode": "train", "epoch": 17}
{"loss": 0.6643201112747192, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6587227582931519, "acc": 0.6263967752456665, "length": 1.0000317096710205, "mode": "train", "epoch": 18}
{"loss": 0.6642299294471741, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.658820629119873, "acc": 0.6260546445846558, "length": 1.0000298023223877, "mode": "train", "epoch": 19}
{"loss": 0.6669952869415283, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6595427393913269, "acc": 0.6267388463020325, "length": 1.0000184774398804, "mode": "train", "epoch": 20}
{"loss": 0.6649783253669739, "acc": 0.6230902075767517, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6582478880882263, "acc": 0.6258836984634399, "length": 1.0000163316726685, "mode": "train", "epoch": 21}
{"loss": 0.6626952290534973, "acc": 0.6230902075767517, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6581282615661621, "acc": 0.6257697343826294, "length": 1.0000085830688477, "mode": "train", "epoch": 22}
{"loss": 0.6677187085151672, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.660097599029541, "acc": 0.6218927502632141, "length": 1.0000090599060059, "mode": "train", "epoch": 23}
{"loss": 0.6688418388366699, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6591647267341614, "acc": 0.6258836388587952, "length": 1.0000038146972656, "mode": "train", "epoch": 24}
{"loss": 0.6639191508293152, "acc": 0.6241319179534912, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6583637595176697, "acc": 0.6262259483337402, "length": 1.0000258684158325, "mode": "train", "epoch": 25}
{"loss": 0.6635818481445312, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6580196022987366, "acc": 0.6256556510925293, "length": 1.0000019073486328, "mode": "train", "epoch": 26}
{"loss": 0.6648494005203247, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6579965949058533, "acc": 0.6269098520278931, "length": 1.0000015497207642, "mode": "train", "epoch": 27}
{"loss": 0.6645264029502869, "acc": 0.6234375834465027, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6576123237609863, "acc": 0.6254275441169739, "length": 1.0000007152557373, "mode": "train", "epoch": 28}
{"loss": 0.663270115852356, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6580390930175781, "acc": 0.6248575448989868, "length": 1.0000005960464478, "mode": "train", "epoch": 29}
{"loss": 0.6644942760467529, "acc": 0.6196179986000061, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6588573455810547, "acc": 0.6262257099151611, "length": 1.0000005960464478, "mode": "train", "epoch": 30}
{"loss": 0.6648382544517517, "acc": 0.6220486164093018, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.656717836856842, "acc": 0.625598669052124, "length": 1.0000040531158447, "mode": "train", "epoch": 31}
{"loss": 0.6650581359863281, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6596100926399231, "acc": 0.6255416870117188, "length": 1.0000009536743164, "mode": "train", "epoch": 32}
{"loss": 0.6639688014984131, "acc": 0.6243054866790771, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.6580969095230103, "acc": 0.6269099712371826, "length": 1.000001311302185, "mode": "train", "epoch": 33}
{"loss": 0.6649010181427002, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6571856141090393, "acc": 0.62599778175354, "length": 1.0, "mode": "train", "epoch": 34}
{"loss": 0.6645286083221436, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6598511338233948, "acc": 0.6248003840446472, "length": 1.0000001192092896, "mode": "train", "epoch": 35}
{"loss": 0.6651360988616943, "acc": 0.6210069060325623, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6581116318702698, "acc": 0.6259407997131348, "length": 1.0, "mode": "train", "epoch": 36}
{"loss": 0.6637561321258545, "acc": 0.6237847208976746, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6573299169540405, "acc": 0.6262826919555664, "length": 1.0, "mode": "train", "epoch": 37}
{"loss": 0.6638917922973633, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6565337777137756, "acc": 0.6270240545272827, "length": 1.0000040531158447, "mode": "train", "epoch": 38}
{"loss": 0.6632587909698486, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6572628617286682, "acc": 0.6258836388587952, "length": 1.0, "mode": "train", "epoch": 39}
{"loss": 0.6649746894836426, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6571335792541504, "acc": 0.6256555318832397, "length": 1.000000238418579, "mode": "train", "epoch": 40}
{"loss": 0.6639227271080017, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6579246520996094, "acc": 0.6253705024719238, "length": 1.0, "mode": "train", "epoch": 41}
{"loss": 0.664652407169342, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6575460433959961, "acc": 0.6258836388587952, "length": 1.0, "mode": "train", "epoch": 42}
{"loss": 0.6643372774124146, "acc": 0.6230903267860413, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6578237414360046, "acc": 0.6254845261573792, "length": 1.0, "mode": "train", "epoch": 43}
{"loss": 0.6662150025367737, "acc": 0.6227430105209351, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6584113240242004, "acc": 0.626339852809906, "length": 1.0, "mode": "train", "epoch": 44}
{"loss": 0.6634557247161865, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6568358540534973, "acc": 0.6256555914878845, "length": 1.0, "mode": "train", "epoch": 45}
{"loss": 0.6633950471878052, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6562194228172302, "acc": 0.6263397336006165, "length": 1.0, "mode": "train", "epoch": 46}
{"loss": 0.6655693650245667, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6577790975570679, "acc": 0.6269669532775879, "length": 1.0, "mode": "train", "epoch": 47}
{"loss": 0.6652371883392334, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6572205424308777, "acc": 0.6264537572860718, "length": 1.0, "mode": "train", "epoch": 48}
{"loss": 0.6648146510124207, "acc": 0.6234375834465027, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6571682095527649, "acc": 0.6261118054389954, "length": 1.0, "mode": "train", "epoch": 49}
{"loss": 0.6646498441696167, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6566428542137146, "acc": 0.6257696747779846, "length": 1.0, "mode": "train", "epoch": 50}
{"loss": 0.6645761728286743, "acc": 0.621874988079071, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6569817662239075, "acc": 0.6256555914878845, "length": 1.0, "mode": "train", "epoch": 51}
{"loss": 0.6640707850456238, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6568990349769592, "acc": 0.6262826323509216, "length": 1.0, "mode": "train", "epoch": 52}
{"loss": 0.6649864315986633, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.6565895080566406, "acc": 0.6265679001808167, "length": 1.0, "mode": "train", "epoch": 53}
{"loss": 0.6652226448059082, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6565943956375122, "acc": 0.6254845857620239, "length": 1.0, "mode": "train", "epoch": 54}
{"loss": 0.6641891002655029, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6566973924636841, "acc": 0.6255986094474792, "length": 1.0, "mode": "train", "epoch": 55}
{"loss": 0.665158748626709, "acc": 0.6237847805023193, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6566883325576782, "acc": 0.6258836388587952, "length": 1.0, "mode": "train", "epoch": 56}
{"loss": 0.6641741394996643, "acc": 0.6232638955116272, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6570380926132202, "acc": 0.6253705024719238, "length": 1.0, "mode": "train", "epoch": 57}
{"loss": 0.6663196086883545, "acc": 0.621874988079071, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6573489308357239, "acc": 0.6251996159553528, "length": 1.0, "mode": "train", "epoch": 58}
{"loss": 0.6642275452613831, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6577418446540833, "acc": 0.6254276037216187, "length": 1.0, "mode": "train", "epoch": 59}
{"loss": 0.6680171489715576, "acc": 0.6236110925674438, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6571102738380432, "acc": 0.6258838176727295, "length": 1.0, "mode": "train", "epoch": 60}
{"loss": 0.6644954681396484, "acc": 0.6239582300186157, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.6370370388031006
